backend/src/bundles/hard-skills/hard-skills.repository.ts

```diff
@@ -0,0 +1,35 @@
+import { HardSkillsEntity } from '~/bundles/hard-skills/hard-skills.entity.js';
+import { type HardSkillsModel } from '~/bundles/hard-skills/hard-skills.model.js';
+import { ErrorMessages } from '~/common/enums/enums.js';
+import { type Repository } from '~/common/types/repository.type.js';
+
+class HardSkillsRepository implements Repository {
+    private hardSkillsModel: typeof HardSkillsModel;
+
+    public constructor(hardSkillsModel: typeof HardSkillsModel) {
+        this.hardSkillsModel = hardSkillsModel;
+    }
+
+    public async findAll(): Promise<HardSkillsEntity[]> {
+        const skills = await this.hardSkillsModel.query().execute();
+        return skills.map((it) => HardSkillsEntity.initialize(it));
```

Comment:
```suggestion
        return skills.map(HardSkillsEntity.initialize);
```

Reply:
Do not pass function directly to `.map(…)`.eslint
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/92392661/9d5ffe94-8461-4cc6-bf57-dda044882860)